spec/lib/shanty_properties_file_plugin/encrypted_properties_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'fileutils'
+require 'gpgme'
+require 'shanty_properties_file_plugin/encrypted_properties'
+require 'spec_helper'
+
+# Tests for properties class
+module PropertiesFilePlugin
+  RSpec.describe(EncryptedProperties) do
+    include_context('properties')
+
+    # Set GPG engine to GPG v1 for testing
+    GPGME::Engine.set_info(0, `which gpg`.strip, nil)
```

Comment:
Could I be a pain and get you to add a comment saying ^^ this just above this line?